tests/serial/watch-integral/WatchIntegralScaleAndNoScale.cpp

```diff
@@ -108,11 +108,11 @@ BOOST_AUTO_TEST_CASE(WatchIntegralScaleAndNoScale)
 
     {
       std::string fileName = "precice-SolverTwo-watchintegral-WatchIntegralNoScale.log";
-      auto        result   = readDoublesFromTXTFile(fileName, 4);
+      auto        result   = readDoublesFromTXTFile(fileName, 3);
       auto        expected = std::vector<double>{
-          1.0, 9.0, 0.0, 3.0,
-          2.0, 12.0, 0.0, 3.0,
-          3.0, 12.0, 0.0, 3.0};
+          1.0, 9.0, 3.0,
+          2.0, 12.0, 3.0,
+          3.0, 12.0, 3.0};
```

Comment:
```suggestion
          1.0, 6.0, 3.0,
          2.0, 9.0, 3.0,
          3.0, 12.0, 3.0};
```
... is what I would actually expect.

Reply:
I opened an issue: https://github.com/precice/precice/issues/1579